lib/dependencies/HarmonyExportImportedSpecifierDependency.js

```diff
@@ -146,7 +147,7 @@ class HarmonyExportImportedSpecifierDependency extends NullDependency {
 
 	getHashValue(importedModule) {
 		if(!importedModule) {
-			return importedModule;
+			return WebpackMissingModule.module(importedModule);
```

Comment:
Here we go. Empty string did the trick as well. Didn't try that option so wasn't aware a missing module error will be issued elsewhere.